compiler/circle-execution-plan/src/CircleExecutionPlan.cpp

```diff
@@ -82,7 +115,7 @@ int entry(int argc, char **argv)
   auto module = importer.importModule(circle_model);
 
   // Do main job
-  circle_planner::ExecutionPlanner execution_planner(module->graph());
+  circle_planner::ExecutionPlanner execution_planner(module->graph(), platform_type, use_dsp);
```

Comment:
What about make "TargetPlatform" structure/class and add `platform_type` and `use_dsp` there?
I think at some point we will need even more options, so this function definition will grow...